.github/workflows/matrix.js

```diff
@@ -8,28 +8,28 @@ matrix.addAxis({
   title: x => x.version + ', ' + x.group,
   values: [
     // Zulu
-    {group: 'Zulu', version: '8', distribution: 'zulu', jit: 'hotspot'},
     {group: 'Zulu', version: '11', distribution: 'zulu', jit: 'hotspot'},
     {group: 'Zulu', version: '16', distribution: 'zulu', jit: 'hotspot'},
```

Comment:
You can remove java16 as it doesn't have any support https://en.wikipedia.org/wiki/Java_version_history